config/credentials.yml.enc

```diff
@@ -1 +1 @@
-k+EJj7YkRhdigyIFVPD9Q0S+KSoBjWRsbw2zXo4QUevN56tJhs+s83rg9RuM45VpBTYIB3OB0bLMR9tluefZJx1Uw0sjii0mzY7yDlIBJWiO6ZQIwhZye6BPoYxUgajjpHkKaDiHHNPc23yitzumHC0EITYhgU1G+n5CLGRDgrlvLrrW/CGKmsFMCLYfDLZqeE+NGlHUhHumRolTTWVEI+TU9VPA/4YepkxtGdbBbp9Hq0fSI3C4PPihGhvE3MrpLK9V4EU+/UiHgurBZh9fb4dL039RdJioTueQKNLYnuv9z83wRV67PfOc8/xsbLLXamz5JpeJmi9ejuWzgTepPU074k9+72iLYRfgEstuFAPYHraXfx37q8BhTYWqexCJbmEbwY4XU/oX1lER4XH6B5GdCJAAVonJxWP0--OtrmvJxS1bOv5uKC--fVKvg8e6GW5sbAQQP8pDEA==
\ No newline at end of file
+k+EJj7YkRhdigyIFVPD9Q0S+KSoBjWRsbw2zXo4QUevN56tJhs+s83rg9RuM45VpBTYIB3OB0bLMR9tluefZJx1Uw0sjii0mzY7yDlIBJWiO6ZQIwhZye6BPoYxUgajjpHkKaDiHHNPc23yitzumHC0EITYhgU1G+n5CLGRDgrlvLrrW/CGKmsFMCLYfDLZqeE+NGlHUhHumRolTTWVEI+TU9VPA/4YepkxtGdbBbp9Hq0fSI3C4PPihGhvE3MrpLK9V4EU+/UiHgurBZh9fb4dL039RdJioTueQKNLYnuv9z83wRV67PfOc8/xsbLLXamz5JpeJmi9ejuWzgTepPU074k9+72iLYRfgEstuFAPYHraXfx37q8BhTYWqexCJbmEbwY4XU/oX1lER4XH6B5GdCJAAVonJxWP0--OtrmvJxS1bOv5uKC--fVKvg8e6GW5sbAQQP8pDEA==
```

Comment:
Is this file still necessary while you are using .env?

Reply:
deleted